test-requirements.txt

```diff
@@ -11,4 +11,4 @@ flake8==3.5.0
 six==1.11.0
 twine==1.11.0
 pytest==4.6.4
-Pillow==6.2.2
+Pillow==7.2.0
```

Comment:
```suggestion
Pillow==6.2.2; python_version < '3.5'
Pillow==7.2.0; python_version >= '3.5'
```

Reply:
I don't know why I forgot you can pin versions, but yes this is the most correct approach.